### PR TITLE
bump version

### PR DESCRIPTION
**Pull request by dr-bonez**

### CODE REVIEW

File: image-recipe/build.sh [hunk 1]
```diff
@@ -158,8 +158,8 @@ echo "deb [arch=${IB_TARGET_ARCH} signed-by=/etc/apt/trusted.gpg.d/tor.key.gpg]
 curl -fsSL https://download.docker.com/linux/debian/gpg | gpg --dearmor -o config/archives/docker.key
 echo "deb [arch=${IB_TARGET_ARCH} signed-by=/etc/apt/trusted.gpg.d/docker.key.gpg] https://download.docker.com/linux/debian ${IB_SUITE} stable" > config/archives/docker.list
 
-curl -fsSL https://download.opensuse.org/repositories/devel:kubic:libcontainers:unstable/Debian_Testing/Release.key | gpg --dearmor -o config/archives/podman.key
-echo "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/trusted.gpg.d/podman.key.gpg] https://download.opensuse.org/repositories/devel:kubic:libcontainers:unstable/Debian_Testing/ /" > config/archives/podman.list
+curl -fsSL https://http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/Debian_12/Release.key | gpg --dearmor -o config/archives/podman.key
+echo "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/trusted.gpg.d/podman.key.gpg] https://http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/Debian_12/ /" > config/archives/podman.list
```
Comment (dr-bonez) [inline image-recipe/build.sh:162]:
```suggestion
echo "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/trusted.gpg.d/podman.key.gpg] https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/Debian_12/ /" > config/archives/podman.list
```

**Review by Blu-J — DISMISSED** [review]

**Review by Blu-J — APPROVED** [review]

File: image-recipe/build.sh [hunk 2]
```diff
@@ -158,8 +158,8 @@ echo "deb [arch=${IB_TARGET_ARCH} signed-by=/etc/apt/trusted.gpg.d/tor.key.gpg]
 curl -fsSL https://download.docker.com/linux/debian/gpg | gpg --dearmor -o config/archives/docker.key
 echo "deb [arch=${IB_TARGET_ARCH} signed-by=/etc/apt/trusted.gpg.d/docker.key.gpg] https://download.docker.com/linux/debian ${IB_SUITE} stable" > config/archives/docker.list
 
-curl -fsSL https://download.opensuse.org/repositories/devel:kubic:libcontainers:unstable/Debian_Testing/Release.key | gpg --dearmor -o config/archives/podman.key
-echo "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/trusted.gpg.d/podman.key.gpg] https://download.opensuse.org/repositories/devel:kubic:libcontainers:unstable/Debian_Testing/ /" > config/archives/podman.list
+curl -fsSL https://http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/Debian_12/Release.key | gpg --dearmor -o config/archives/podman.key
```
Comment (GlenCooper) [inline image-recipe/build.sh]: This looks incorrect to me.  Is it?  Please verify.
"https://http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/Debian_12/Release.key" seems like an incorrectly formatted URL.

File: image-recipe/build.sh (hunk 1 above)
Comment (GlenCooper) [inline image-recipe/build.sh]: This looks incorrect to me. Is it? Please verify.
"https://http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/Debian_12/" seems like an incorrectly formatted URL.

**Review by GlenCooper — COMMENTED** [review]
looks like incorrectly formatted URLs

**Pull request merged**